lib/capture/streamer/mock.js

```diff
@@ -0,0 +1,81 @@
+const { Readable } = require('node:stream')
+const fs = require('fs')
+const path = require('path')
+const sharp = require('sharp')
+
+const DEFAULT_TIMEOUT = 180_000
+const IMAGE_PATH = path.join(__dirname, '../../../mock_data/compliance/mock.jpg')
+
+const stream = (options = {}) => {
+  const { timeout = DEFAULT_TIMEOUT } = options
+  
+  const readableStream = new Readable({
+    objectMode: true,
+    read() {}
+  })
+
+  // Mock image path
+  const mockImagePath = path.join(IMAGE_PATH)
+  
+  if (!fs.existsSync(mockImagePath)) {
```

Comment:
Probably unnecessary, an error would be thrown later in `initialize()` if the file doesn't exist anyway (plus, it's a classic check before use race condition).